src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -199,7 +201,8 @@ public boolean getUseHttp() {
     }
 
     @DataBoundSetter
-    public void setUseHttp(boolean useHttp){
+    public void setUseHttp(boolean useHttp) throws IllegalArgumentException {
```

Comment:
```suggestion
    public void setUseHttp(boolean useHttp) {
```
Runtime exceptions don't need to be declared.
Also, regardless we're using either `FormException` or `IllegalArgumentException` to indicate FIPS non compliance errors, I'm ok with launching a `Failure`, it gives the information and avoids saving wrong values, so no problem with that